test/integration/proxy/proxy.go

```diff
@@ -0,0 +1,126 @@
+// +build integration
+
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package proxy
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/elazarl/goproxy"
+	minishiftUtil "github.com/minishift/minishift/pkg/minishift/util"
+)
+
+var (
+	proxyListener net.Listener
+	proxyLog      *bytes.Buffer
+)
+
+func GetLog() string {
+	var log string
+	if proxyLog != nil {
+		log = proxyLog.String()
+	} else {
```

Comment:
Else part not needed. Zero/default value of string is empty. See https://tour.golang.org/basics/12.